platform/ios/src/MGLScaleBar.h

```diff
@@ -6,4 +6,10 @@
 // Sets the scale and redraws the scale bar
 @property (nonatomic, assign) CLLocationDistance metersPerPoint;
 
+// Expose for testing
```

Comment:
Have you tried forward declaration in the test case instead?

Reply:
Done.